lib/rules/function-url-no-scheme-relative/index.js

```diff
@@ -25,7 +24,7 @@ function rule(actual) {
 
 		root.walkDecls((decl) => {
 			functionArgumentsSearch(decl.toString().toLowerCase(), 'url', (args, index) => {
-				const url = _.trim(args, ' \'"');
+				const url = args.replace(/^[ '"]+|[ '"]+$/g, '');
```

Comment:
[question] Does it make sense to use a combination of the native `trim()` and a regex as below?

```suggestion
				const url = args.trim().replace(/^['"]+|['"]+$/g, '');
```

Because the native `trim()` removes not only `' '` but also other whitespace characters like `\t` or `\uFEFF`.
See the [polyfill on MDN](http://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim#polyfill).

Reply:
Yes, I think that makes sense. Also it matches with the behavior of the other code changed by this PR.